src/plugin/ClickableTaskLabelPlugin.js

```diff
@@ -97,6 +98,10 @@ export class ClickableTaskLabelPlugin extends PluginInterface {
       stories: this.roadmappy.roadmap.getStories().map(s => s.toAssoc()),
       assignees: this.roadmappy.roadmap.getAssignees().map(a => a.toAssoc()),
     });
+
+    this.form.style.left = `${d3.event.offsetX + 16}px`;
```

Comment:
I think should to route mouse position without `d3.event.offsetX`
For example, in `RoadmapCanvas.js`
```
.on('click', (d) => this.emit('click:task-label', d, d3.event.target, d3.mouse(this.svg.node())[0], d3.mouse(this.svg.node())[1]))
```